### PR TITLE
Done the getting a playone list

### PR DESCRIPTION
**Pull request by pokk**
Finished retrieving a playone list from the remote layer thru until presetation layer.

### CODE REVIEW

**Comment by pokk** [comment]
@lekaha 

Please take a look. Thanks!

File: domain/src/main/java/com/playone/mobile/domain/interactor/playone/GetPlayoneList.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * instances from the [com.playone.mobile.domain.repository.PlayoneRepository]
+ */
+open class GetPlayoneList constructor(
+    private val repository: PlayoneRepository,
+    threadExecutor: ThreadExecutor,
+    postExecutionThread: PostExecutionThread
+) : SingleUseCase<List<Playone>, Int?>(threadExecutor, postExecutionThread) {
+
+    public override fun buildUseCaseObservable(params: Int?): Single<List<Playone>> =
+        repository.getPlayoneList(params ?: -1)
```
Comment (lekaha) [inline domain/src/main/java/com/playone/mobile/domain/interactor/playone/GetPlayoneList.kt]: If `params` is not allowed being null you should throw runtime exception I think

Comment (lekaha) [reply domain/src/main/java/com/playone/mobile/domain/interactor/playone/GetPlayoneList.kt]: Don't just pass a meaningless (magic) number

File: mobile-ui/src/dependant/java/com/playone/mobile/ui/firebase/v1/PlayoneFirebaseV1.kt [hunk 2]
```diff
@@ -33,7 +33,7 @@ class PlayoneFirebaseV1(
         userId: Int,
         callback: PlayoneCallback<List<PlayoneModel>>,
         errorCallback: FirebaseErrorCallback
-    ) = if (0 < userId) {
+    ) = if (0 > userId) {
         playoneDsAction(callback, errorCallback, ::snapToPlayoneList)
```
Comment (lekaha) [inline mobile-ui/src/dependant/java/com/playone/mobile/ui/firebase/v1/PlayoneFirebaseV1.kt:38]: So I think if you solve the last view this statement is not needed.

**Review by lekaha — CHANGES_REQUESTED** [review]

**Review by lekaha — APPROVED** [review]

**Pull request merged**